### ISSUE TITLE
Setting of the hostname doesn't work on 2022-01-28-raspios-bullseye-armhf-lite

### ISSUE THREAD

**Issue by polac** [acimadamore/ansible-role-raspberry-pi#2]
Disclaimer: I just started learning ansible.

Thanks for the awesome role. I installed 2022-01-28-raspios-bullseye-armhf-lite-image on my Raspberry PI 4 and tried to run your role. 

It was unable to set the hostname
```
The full traceback is:                                                                                                                                                                                         
Traceback (most recent call last):                                                                                                                                                                             
  File "/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py", line 107, in <module>                                                                              
    _ansiballz_main()                                                                                                                                                                                          
  File "/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py", line 99, in _ansiballz_main                                                                        
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)                                                                                                                                                     
  File "/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py", line 47, in invoke_module                                                                          
    runpy.run_module(mod_name='ansible.modules.hostname', init_globals=dict(_module_fqn='ansible.modules.hostname', _modlib_path=modlib_path),                                                                 
  File "/usr/lib/python3.9/runpy.py", line 210, in run_module                                                                                                                                                  
    return _run_module_code(code, init_globals, run_name, mod_spec)                                                                                                                                            
  File "/usr/lib/python3.9/runpy.py", line 97, in _run_module_code                                                                                                                                             
    _run_code(code, mod_globals, init_globals,                                                                                                                                                                 
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code                                                                                                                                                    
    exec(code, run_globals)                                                                                                                                                                                    
  File "/tmp/ansible_hostname_payload_mz4t44qx/ansible_hostname_payload.zip/ansible/modules/hostname.py", line 888, in <module>                                                                                
  File "/tmp/ansible_hostname_payload_mz4t44qx/ansible_hostname_payload.zip/ansible/modules/hostname.py", line 857, in main                                                                                    
  File "/tmp/ansible_hostname_payload_mz4t44qx/ansible_hostname_payload.zip/ansible/modules/hostname.py", line 162, in __init__                                                                                
KeyError: 'DebianStrategy'                                                                                                                                                                                     
fatal: [192.168.66.109]: FAILED! => {                                                                                                                                                                          
    "changed": false,                                                                                  
    "module_stderr": "Shared connection to 192.168.66.109 closed.\r\n",                                                                                                                                        
    "module_stdout": "Traceback (most recent call last):\r\n  File \"/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py\", line 107, in <module>\r\n    _ansibal
lz_main()\r\n  File \"/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py\", line 99, in _ansiballz_main\r\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PA
RAMS)\r\n  File \"/home/pi/.ansible/tmp/ansible-tmp-1648034130.3562624-365285-140478778528755/AnsiballZ_hostname.py\", line 47, in invoke_module\r\n    runpy.run_module(mod_name='ansible.modules.hostname', i
nit_globals=dict(_module_fqn='ansible.modules.hostname', _modlib_path=modlib_path),\r\n  File \"/usr/lib/python3.9/runpy.py\", line 210, in run_module\r\n    return _run_module_code(code, init_globals, run_n
ame, mod_spec)\r\n  File \"/usr/lib/python3.9/runpy.py\", line 97, in _run_module_code\r\n    _run_code(code, mod_globals, init_globals,\r\n  File \"/usr/lib/python3.9/runpy.py\", line 87, in _run_code\r\n  
  exec(code, run_globals)\r\n  File \"/tmp/ansible_hostname_payload_mz4t44qx/ansible_hostname_payload.zip/ansible/modules/hostname.py\", line 888, in <module>\r\n  File \"/tmp/ansible_hostname_payload_mz4t44
qx/ansible_hostname_payload.zip/ansible/modules/hostname.py\", line 857, in main\r\n  File \"/tmp/ansible_hostname_payload_mz4t44qx/ansible_hostname_payload.zip/ansible/modules/hostname.py\", line 162, in __
init__\r\nKeyError: 'DebianStrategy'\r\n",                                                                                                                                                                     
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1                                                                                            
}        
```

This seems to be known problem. Please see: https://github.com/ansible/ansible/issues/76124

If I have understand correctly this will be fixed on ansible-core release 2.13.

While we wait for that we could fix it also by setting the strategy to systemd as I have done in PR #1

**Comment by acimadamore** [comment]
Hi @polac sorry for the late response. I think i had the same problem. I removed the strategy completely and it worked but i forgot to commit that change sorry! I will look into this and your PR as soon as i can. 

Thanks for using this role c:

**Comment by acimadamore** [comment]
Fixed at PR #1. Thanks @polac !
